src/storage/src/hummock/sstable/builder.rs

```diff
@@ -52,6 +53,7 @@ impl Default for SSTableBuilderOptions {
     }
 }
 
+pub const BITMAP_LEN: usize = 1 << ((VALUE_META_SIZE << 3) - 3);
```

Comment:
Could define another constant for the `u16` vnode value, in case more fields be added to value meta in the future (`VALUE_META_SIZE ` is not the same as the size of `u16` vnode value).

Reply:
Also suggest rename `BITMAP_LEN` to `VNODE_BITMAP_LEN` to be more specific, since it is `pub` to outside.

Reply:
fixed

Reply:
> Also suggest rename `BITMAP_LEN` to `VNODE_BITMAP_LEN` to be more specific, since it is `pub` to outside.

fixed